orchestrator/careplanservice/subscriptions/manager.go

```diff
@@ -75,22 +75,15 @@ func (r DerivingManager) Notify(ctx context.Context, resource interface{}) error
 			Type:      to.Ptr("CarePlan"),
 		}
 
-		for _, team := range carePlan.CareTeam {
-			if team.Reference == nil {
-				continue
-			}
-
-			var careTeam fhir.CareTeam
-
-			if err := r.FhirClient.ReadWithContext(ctx, *team.Reference, &careTeam); err != nil {
-				log.Ctx(ctx).Error().Msgf("failed to read CareTeam in FHIR store %s: %s", *team.Reference, err)
-				continue
-			}
+		careTeam, err := coolfhir.ResolveCareTeam(carePlan)
+		if err != nil {
+			log.Ctx(ctx).Error().Msgf("failed to read CareTeam in CarePlan %s", err)
```

Comment:
have the error populate the structurized logging field meant for it:
```suggestion
			log.Ctx(ctx).Err(err).Msg("failed to read CareTeam in CarePlan")
```